xpallets/support/src/macros.rs

```diff
@@ -56,93 +54,3 @@ macro_rules! try_hex {
         &$x
     }};
 }
-
-#[cfg(feature = "std")]
-#[macro_export]
-macro_rules! ensure_with_errorlog {
-    ( $x:expr, $y:expr, $($arg:tt)*) => {{
-        if !$x {
-            $crate::error!("{:?}|{}", $y, format!($($arg)*));
-            $crate::fail!($y);
-        }
-    }}
-    }
-
-#[cfg(not(feature = "std"))]
-#[macro_export]
-macro_rules! ensure_with_errorlog {
-    ( $x:expr, $y:expr, $($arg:tt)*) => {{
-        if !$x {
-            $crate::fail!($y);
-        }
-    }};
-}
-
-pub mod log {
-    pub const RUNTIME_TARGET: &str = "runtime";
-
-    #[macro_export]
-    macro_rules! error {
-        (target: $target:expr, $($arg:tt)+) => (
-            frame_support::debug::error!(target: $target, $($arg)+);
-        );
-        ($($arg:tt)+) => (
-            #[cfg(feature = "std")]
-            $crate::error!(target: &format!("runtime::{}:{}", module_path!(), line!()), $($arg)+);
```

Comment:
These log marco wrappers are not _useless_ as it will display some extra info like the runtime module & line number in the target field under `std`.